perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -281,12 +282,15 @@ sub category_extras_ajax : Path('category_extras') : Args(0) {
     my $councils_text_private = $c->render_fragment( 'report/new/councils_text_private.html');
 
     $unresponsive = $c->stash->{unresponsive}->{$category} || $c->stash->{unresponsive}->{ALL} || '';
+
+    my @bodies = map { $_->name } @$bodies;
```

Comment:
And here also?

Reply:
This is only ever used in a json object so I don't think it needs the CSV handling because were not splitting it.

Reply:
Good point!